storage/ProcessDatabaseSchema.ts

```diff
@@ -26,88 +26,136 @@ type ProcessDatabaseSchema = NamedSchema<
         };
         rooms: {
           canEnterAll: string;
-          notes: string;
+          notes: {
+            value: string;
```

Comment:
I tried to run this locally and saw some errors in the console. After a discussion with @erbridge we realised that we need to add a migration for the new `notes` field on each property. 

I've been informed you're being pulled onto other projects @LBHELewis and may be short on time, so I've made a start on trying to fix this, but it's currently very much a [work in progress](https://github.com/LBHackney-IT/mat-process-thc/commit/a54ef9de655e7b72995944d4f285c79ec62ec607).

I'll keep going with it and clear up any issues, but just wanted to keep you posted!

Reply:
@Gweaton Amazing, thank you George.